ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -89,6 +89,11 @@ def getFTS3Servers():
   return S_OK( ftsServers )
 
 
+def getFTS3ServerDict():
+  """:returns: dict of key = server name and value = server url
```

Comment:
@andresailer will it work without new line there ?

Reply:
I am not sure,.. Anyway would be better to have a short description docstring a newline and the the `:returns:` statement.
But I think none of the FTS3 code documentation is currently building because of missing dependencies.

Reply:
maybe something that @hamzazafar can check in his next PR :)